prom2json_test.go

```diff
@@ -47,6 +45,17 @@ var tcs = []testCase{
 						Value: floatPtr(1),
 					},
 				},
+				// Test metric with tags and timestamp
+				&dto.Metric{
+					Label: []*dto.LabelPair{
+						createLabelPair("tag1", "foo"),
+						createLabelPair("tag2", "bar"),
+					},
+					TimestampMs: intPtr(123456),
```

Comment:
You can use proto.Int64 for this.

Reply:
I know, but I wanted to stay consistent with how it is already done here for other types (some of which could also have done with `proto.Xxx`).